src/EnergyPlus/AirflowNetwork/include/AirflowNetwork/Solver.hpp

```diff
@@ -45,109 +45,317 @@
+        int SupplyFanType = 0;                   // Supply air fan type
+        Real64 MaxOnOffFanRunTimeFraction = 0.0; // max Run time fraction for an On/Off fan flow rate among airloops
+        Real64 CurrentEndTimeLast = 0.0;         // last end time
+        Real64 TimeStepSysLast = 0.0;            // last system time step
+        int AirflowNetworkNumOfOccuVentCtrls = 0;
+        int IntraZoneNumOfNodes = 0;
+        int IntraZoneNumOfLinks = 0;
+        int IntraZoneNumOfZones = 0;
+        int NumOfPressureControllers = 0; // number of pressure controllers
+        int NumOfOAFans = 0;              // number of OutdoorAir fans
+        int NumOfReliefFans = 0;          // number of OutdoorAir relief fans
+        bool AirflowNetworkGetInputFlag = true;
+        bool AssignFanAirLoopNumFlag = true;
+        bool ValidateDistributionSystemFlag = true;
+        Array1D<Real64> FacadeAng =
+            Array1D<Real64>(5); // Facade azimuth angle (for walls, angle of outward normal to facade measured clockwise from North) (deg)
+        Array1D<Real64> LoopPartLoadRatio;
+        Array1D<Real64> LoopOnOffFanRunTimeFraction;
+        Array1D<bool> LoopOnOffFlag;
+
+        bool ValidateExhaustFanInputOneTimeFlag = true;
+        bool initializeOneTimeFlag = true;
+        bool initializeMyEnvrnFlag = true;
+        bool CalcAirflowNetworkAirBalanceOneTimeFlag = true;
+        bool CalcAirflowNetworkAirBalanceErrorsFound = false;
+        bool UpdateAirflowNetworkMyOneTimeFlag = true;
+        bool UpdateAirflowNetworkMyOneTimeFlag1 = true;
+
+        // CalcAirflowNetworkAirBalance variables
+        int ErrCountVar = 0;
+        int ErrCountHighPre = 0;
+        int ErrCountLowPre = 0;
+        int ErrIndexHighPre = 0;
+        int ErrIndexVar = 0;
+        int ErrIndexLowPre = 0;
+
+        // Object Data
+        EPVector<AirflowNetwork::AirflowNetworkReportVars> AirflowNetworkZnRpt;
+        std::unordered_map<std::string, std::string> UniqueAirflowNetworkSurfaceName;
+
+        // Output and reporting
+        EPVector<AirflowNetwork::AirflowNetworkExchangeProp> exchangeData;
+        EPVector<AirflowNetwork::AirflowNetworkExchangeProp> multiExchangeData;
+        EPVector<AirflowNetwork::AirflowNetworkLinkReportData> linkReport;
+        EPVector<AirflowNetwork::AirflowNetworkNodeReportData> nodeReport;
+        EPVector<AirflowNetwork::AirflowNetworkLinkReportData> linkReport1;
+
+        // used to be statics
+        Array1D<bool> onceZoneFlag;
+        Array1D<bool> onceSurfFlag;
+        bool onetime = false;
+        int HybridGlobalErrIndex = 0;
+        int HybridGlobalErrCount = 0;
+        int AFNNumOfExtOpenings = 0; // Total number of external openings in the model
+        int OpenNuminZone = 0;       // Counts which opening this is in the zone, 1 or 2
 
         std::unordered_map<std::string, AirflowElement *> elements;
```

Comment:
This bad girl I'm less sure about.

Reply:
This is the one I'm sure about.  For historical reasons, models may have a very large number of elements (could be on the order of the number of surfaces), so there could be a lot of these (100s to 1000s). It is only used during setup to map an element name to the element that is needed.